cypress/integration/splitButton.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Split Button component
+  I want to change Split Button component properties
+
+  Background: Open Split Button component page classic
+    Given I open "Split Button" component page classic
+
+  @positive
+  Scenario Outline: I select as to <as>
+    When I select as to "<as>"
+    Then Button background color is "<color>"
+    Examples:
+      | as        | color            |
+      | primary   | rgb(37, 91, 199) |
+      | secondary | rgba(0, 0, 0, 0) |
+
+  @positive
+  Scenario Outline: I set data-element to <data-element>
+    When I set data-element to "<data-element>"
+    Then data-element "<data-element>" is present
+    Examples:
+      | data-element            |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+
+  @positive
+  Scenario Outline: I set data-role to <data-role>
+    When I set data-role to "<data-role>"
+    Then data-role "<data-role>" is present
+    Examples:
+      | data-role               |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+
+  @positive
+  Scenario: I disable Split Button component
+    When I disable SplitButton component
+    Then Button is disabled
```

Comment:
Split Button or button ->

Reply:
it is capial letter because step is reused